Makefile

```diff
@@ -56,10 +56,10 @@ else
 endif
 
 build-extensions:
-	$(foreach dir, $(wildcard $(EXTENSIONS_DIR)/*), $(MAKE) -C $(dir) build;)
+	yarn intree-extensions $(EXTENSIONS_DIR) build
```

Comment:
how does yarn find intree-extensions?

Reply:
`yarn <verb>` means roughly the same as `yarn run <verb>` if `<verb>` is not a predefined action in `yarn`. So it looks at the "scripts" object in the `package.json` file at the root of this repo.

Reply:
ah, I missed your change to package.json

Reply:
My initial idea was just to replace this foreach with a call to npm ... is that a bad idea / why we did implement our own logic (which kinda reinvents a wheel)?

I mean something like:
```
$(foreach dir, $(wildcard $(EXTENSIONS_DIR)/*), cd $(dir) &&  npm install && npm build;)
```

Reply:
There is nothing wrong with that. Reverted.